splash/browser_tab.py

```diff
@@ -768,3 +793,64 @@ def log(self, message, min_level=None):
 
         message = "[%s] %s" % (self.uid, message)
         log.msg(message, system='render')
+
+
+class _LuaResume(QObject):
+    """ Allows us to resume a Lua coroutine from a Javascript context. """
+
+    def __init__(self, parent, callback, errback, timeout=0):
+        self.name = _LuaResume.generate_name()
```

Comment:
I think it should be self.generate_name(), otherwise @classmethod decortor is pointless